Backend/utils/ResponseCreation.py

```diff
@@ -0,0 +1,45 @@
+import json
+from flask import Response
+
+
+class ControllerResponse:
+  '''
+  An object to handle managment of
+  the JSON response that we want REST calls
+  to return. 
+  '''
+  def __init__(self):
+    self.errors = []
+    self.payload = {}
+    self.hasError = False
```

Comment:
Curious to why there is a flag here. If len(self.errors) == 0 doesn't this imply that the response has no errors. Conversely if len(self.errors != 0) implies the response has errors?